### ISSUE TITLE
hello:org does not work

### ISSUE THREAD

**Issue by catalandres** [forcedotcom/sfdx-plugin-generate#49]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
Report a bug.

What is the current behavior?
-----------------------------
1. Create a folder somewhere and go into it.
1. Initialize a plugin in the folder, e.g. using `sfdx plugins:generate --defaults`
1. Link the plugin for its use with `sfdx`: `sfdx plugins:link .`
1. Run `sfdx hello:org -u [aliasName]`

The result is:
```
ERROR running Org:  Invalid URI "/services/data/v39.0/query?q=Select%20Name%2C%20TrialExpirationDate%20from%20Organization"
```

What is the expected behavior?
------------------------------
The expected result would be something along the lines of this string:
```
let outputString = `Hello ${name}! This is org: ${orgName}`;
```

Please mention your Salesforce CLI and OS version.

Running on macOS 10.14.2.

```
sfdx --version
sfdx-cli/6.43.1-6021deb31d (darwin-x64) node-v8.9.4
```

```
sfdx plugins --core
@salesforce/plugin-generator 1.0.1 (core)
@salesforce/sfdx-trust 1.0.8 (core)
builtins 1.0.0 (core)
salesforcedx 44.10.0 (core)
test 0.0.0 (link)
```

**Comment by rbergman** [comment]
Thanks for the report.  This was fixed in sfdx-core yesterday.  We expect to publish a new release of it soon.

**Comment by rbergman** [comment]
A new version of sfdx-core has been released (v1.1.0) that should solve this.  You will need to reinstall dependencies in your plugin and update your lock file if you have one.
